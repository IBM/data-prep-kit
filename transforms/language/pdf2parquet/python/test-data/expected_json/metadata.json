{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2024-09-10 21:28:38",
    "end_time": "2024-09-10 21:28:47",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "artifacts_path": null,
    "contents_type": "application/json",
    "do_table_structure": true,
    "do_ocr": true,
    "double_precision": 0,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".zip"
    ],
    "num_processors": 0
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 605137,
    "result_files": 2,
    "result_size": 33316,
    "processing_time": 6.048,
    "nrows": 3,
    "nsuccess": 3,
    "nfail": 0,
    "nskip": 0
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/python/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/python/output",
    "type": "path"
  }
}