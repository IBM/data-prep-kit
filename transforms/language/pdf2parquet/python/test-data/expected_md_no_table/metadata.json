{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2024-07-24 10:10:06",
    "end_time": "2024-07-24 10:10:41",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "artifacts_path": null,
    "contents_type": "text/markdown",
    "do_table_structure": false,
    "do_ocr": false,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".zip"
    ]
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 8925664,
    "result_files": 2,
    "result_size": 81129,
    "processing_time": 23.67374563217163,
    "nrows": 3,
    "nsuccess": 3
  },
  "source": {
    "name": "/data/docling-dev/data-prep-kit/transforms/language/pdf2parquet/python/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/data/docling-dev/data-prep-kit/transforms/language/pdf2parquet/python/output",
    "type": "path"
  }
}