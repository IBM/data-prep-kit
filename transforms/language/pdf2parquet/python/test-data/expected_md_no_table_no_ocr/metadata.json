{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2024-08-22 16:04:27",
    "end_time": "2024-08-22 16:04:42",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "artifacts_path": null,
    "contents_type": "text/markdown",
    "do_table_structure": false,
    "do_ocr": false,
    "double_precision": 0,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".zip"
    ],
    "num_processors": 0
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 605137,
    "result_files": 2,
    "result_size": 28828,
    "processing_time": 10.41,
    "nrows": 3,
    "nsuccess": 3,
    "nfail": 0,
    "nskip": 0
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/python/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/python/output",
    "type": "path"
  }
}