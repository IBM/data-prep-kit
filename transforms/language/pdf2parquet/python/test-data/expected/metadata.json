{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2024-08-22 16:01:48",
    "end_time": "2024-08-22 16:02:07",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "artifacts_path": null,
    "contents_type": "text/markdown",
    "do_table_structure": true,
    "do_ocr": true,
    "double_precision": 0,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".zip"
    ],
    "num_processors": 0
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 605137,
    "result_files": 2,
    "result_size": 31976,
    "processing_time": 12.429,
    "nrows": 3,
    "nsuccess": 3,
    "nfail": 0,
    "nskip": 0
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/python/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/language/pdf2parquet/python/output",
    "type": "path"
  }
}